packages/fluentui/react-northstar/src/components/Table/Table.tsx

```diff
@@ -90,15 +101,14 @@ class Table extends UIComponent<WithAsProp<TableProps>> {
       return TableRow.create(row, {
         defaultProps: () => ({
           ...props,
-          accessibility: accessibility.childBehaviors ? accessibility.childBehaviors.row : undefined,
+          accessibility: (accessibility && accessibility({ header: !!header })?.childBehaviors.row) || undefined,
```

Comment:
Don't forget to add `{...getA11yProps('row', {`

Reply:
Fixed